packages/zql-integration-tests/src/chinook/chinook.pg-test.ts

```diff
@@ -535,6 +535,44 @@ describe(
               },
             };
           })(),
+          // ordering by nullable columns
+          {
+            name: 'Order by nullable column',
+            createQuery: q => q.customer.orderBy('company', 'desc').limit(10),
+          },
+          // TODO: case below is broken.
```

Comment:
this is breakage is unrelated to the current change and is broken on main as well.